src/_includes/banner.html

```diff
@@ -31,7 +31,8 @@
   {%- else -%}
     <p class="banner__text">
       <i class="material-icons"> new_releases </i>
-      This site covers <b>web development</b> with <b>Dart 2</b>.
+      This site covers <b>web development</b> with <b>Dart 2</b>
```

Comment:
Ha, I just realized the **Dart 2** is in there. I think we can drop that. :)

...with Angular 5

Reply:
Done